src/components/Overview.tsx

```diff
@@ -240,7 +240,6 @@ export default class Overview extends Component<OverviewProps, OverviewState> {
 
 	renderHeader = (loadMore: LoadMore) => {
 		return (
-			!loadMore.isEmpty() && (
```

Comment:
I think filter controls should not show on initial empty result - when no filters are set and still no apps.

So I think here should be condition if any filters are set.

Reply:
@ViliamKopecky done